tooling/cli/templates/mobile/ios/project.yml

```diff
@@ -31,6 +31,7 @@ targets:
     sources:
       - path: Sources
       - path: Assets.xcassets
+      - path: Externals
```

Comment:
Can we just use Sources directory above?

Reply:
Since this is hopefuly temporary, I'll keep it in a separate folder.